src/app-layer-smtp.c

```diff
@@ -383,7 +382,6 @@ static void SMTPConfigure(void) {
         } else {
             /* Add default extract url scheme 'http' since
              * extract-urls-schemes wasn't found in the config */
-            SCMimeSmtpConfigExtractUrlsSchemeReset();
```

Comment:
Thanks I think I used this while trying to have a fuzz target that reload the configuration for each input